discovery/marathon/marathon.go

```diff
@@ -57,6 +58,9 @@ const (
 
 	// Constants for instrumentation.
 	namespace = "prometheus"
+
+	// Maximum bytes to read in remote response
+	maxRemoteReadBytes = 10 * 1024 * 1024
```

Comment:
I couldn't find a share spot to define these. I tried `discovery/config` and the root `config` package, but both cause import cycles. 

Do we want them configurable in yaml?